src/content/enterprise/administration/configuration.mdx

```diff
@@ -287,11 +288,30 @@ autoscaler:
   slackWebhook:
 ```
 
+### backend
+
+Common configuration for backend services.
+
+- `pipelineInstaller.image`: Base image for the binaries needed by the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.annotations`: Annotations to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.env`: Environment variables to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.image`: Custom runtime image for the [installer jobs](/docs/enterprise/administration/configuration/#installer). Defaults to [okteto/pipeline-runner](https://github.com/okteto/pipeline-runner).
+- `installer.labels`: Labels to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `image.registry`: Image registry URL for the [api](/docs/enterprise/administration/configuration/#api) and [webhook](/docs/enterprise/administration/configuration/#webhook) services.
```

Comment:
I don't think we should document this, unless we expect this to be changed by the user right?

Reply:
which one? the `image`?
we do expect people to change the image, there are two use cases here:
- companies not pulling from dockerhub
- get access to a commit not released in a chart release yet

Reply:
I removed the `backend` section, we should deprecate it at some point. I moved `labels`, `annotations` and `extraEnv` to the installer section because they are new fields.
The `backend` section is to overwrite our images, we should move that to every component, but for now let's keep it undocumented

Reply:
Sounds good!